it/test/testConstants/IncomeSourceIntegrationTestConstants.scala

```diff
@@ -1138,7 +1138,7 @@ object IncomeSourceIntegrationTestConstants {
         "taxYear" -> "2019",
         "documentId" -> documentId,
         "documentDate" -> "2018-02-14",
-        "documentDescription" -> "ITSA - POA 1",
+        "documentDescription" -> "ITSA- POA 1",
```

Comment:
Please define/use next constants in the test scope:
  private val POA1: String = "ITSA- POA 1"
  private val POA2: String = "ITSA - POA 2"